src/daemon/database/interface.rs

```diff
@@ -253,21 +255,20 @@ impl TryFrom<&Row<'_>> for DbVault {
             deposit_outpoint,
             amount,
             derivation_index,
-            received_at,
-            updated_at,
+            funded_at,
+            secured_at,
+            delegated_at,
+            moved_at,
             final_txid,
         })
     }
 }
 
-/// Get all the vaults we know about from the db, sorted by last update
```

Comment:
DId you check if in revaultd code we expected the vaults to be ordered?

Reply:
this code is used by `listonchaintransactions` and `listvaults` and none stipulated that the vaults needs to be ordered.
This removal in my knowledge does not break anything at the revaultd side and gui does not rely on vault order.
Because of the removal of updated_at, we cannot order a list of vaults with different status in the sql query. We may reintroduce order if needed.